scripts/ssb-web/install

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Install nginx and php
+source "$BASE_DIR/../shared/nginx/install"
+sudo apt-get install -y php-fpm socat
+
+# Install required parts of ssb from npm
+sudo npm install -g --unsafe-perm \
```

Comment:
an alternative here is to install npm with [nvm](https://github.com/creationix/nvm), which allows you to specify which version of npm you'd like to use, and works around needing `sudo` for `-g` packages (they will be run from your user's home directory instead.

If you need npm for multiple users, it may not be suitable out-of-the-box.

Reply:
npm is WIERD. 

I install it globally but a package but it still complains about the package is missing.
i have to link the global repo into the "users" local repo as a work around